.github/scripts/Configure-AppxManifest.ps1

```diff
@@ -49,12 +65,6 @@ elseif ($Branch -eq "SideloadStable")
         (Get-Content $_ -Raw | ForEach-Object -Process { $_ -replace "Assets\\AppTiles\\Dev", "Assets\AppTiles\Release" }) | `
         Set-Content $_ -NoNewline `
     }
-
-    Get-ChildItem $WorkingDir -Include *.cs -recurse | ForEach-Object -Process `
```

Comment:
Was this removed on purpose?

Reply:
This was move to L92 and now uses $Branch val